CONTRIBUTING.md

```diff
@@ -187,6 +187,12 @@ the bots. To update your local flutter version, run:
 
 ```
 
+You also need to generate the required mocks for the tests. To generate the mocks, run:
+```
+cd packages/devtools_test
+dart run build_runner build
```

Comment:
I don't know much about this area but would it be better to have the user run `./tool/refresh.sh`
to make sure that they are regenerating all mocks?
@kenzieschmoll perhaps you know what might be acceptable here.